x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/CertificateToolTests.java

```diff
@@ -368,7 +380,11 @@ public void testGeneratingSignedPemCertificates() throws Exception {
             assertTrue(Files.exists(zipRoot.resolve(filename)));
             final Path cert = zipRoot.resolve(filename + "/" + filename + ".crt");
             assertTrue(Files.exists(cert));
-            assertTrue(Files.exists(zipRoot.resolve(filename + "/" + filename + ".key")));
+            Path keyFile = zipRoot.resolve(filename + "/" + filename + ".key");
+            assertTrue(Files.exists(keyFile));
+            if (keyPassword != null) {
+                assertTrue(Files.readString(keyFile).contains("DEK-Info: AES-128-CBC"));
+            }
```

Comment:
For completeness, can we also
```suggestion
            } else {
               assertFalse(Files.readString(keyFile).contains("DEK-Info:"));
            }
```